### PR TITLE
Add web_embedding sample to sample index

### PR DESCRIPTION
**Pull request by johnpryan**
We should wait until this sample is deployed to https://flutter-angular.web.app/ 
cc: @kevmoo

### CODE REVIEW

**Comment by ditman** [comment]
Sample deployed: https://flutter-angular.web.app

**Review by domesticmouse — APPROVED** [review]

**Comment by domesticmouse** [comment]
Land on green CI. (Well, apart from the `android-build` check)

File: web/samples_index/lib/src/samples.yaml [hunk 1]
```diff
@@ -633,3 +633,23 @@ samples:
     tags: ['demo', 'animation']
     web: https://z.flutter.gallery/#/dice
     type: demo
+
+  - name: Web Embedding
+    author: Flutter
```
Comment (ditman) [inline web/samples_index/lib/src/samples.yaml:638]:
```suggestion
    author: Flutter and Angular
```

Mark Thompson from Angular helped build this, let's give them credit too!

Comment (johnpryan) [reply web/samples_index/lib/src/samples.yaml]: Good catch!

**Comment by johnpryan** [comment]
Merging since verify-web-demos is passing.

**Pull request merged**